packages/fxa-auth-server/lib/routes/index.js

```diff
@@ -173,7 +173,8 @@ module.exports = function (
     customs,
     config.totp,
     glean,
-    profile
+    profile,
+    config.sentry.env
```

Comment:
Why the Sentry env name?

Reply:
It's the most reliable. You can't use NODE_ENV, cause it's production and stage and production...

Reply:
I see.  Maybe update the comment for that config?

Reply:
Added a comment about this in doc for the config setting. (FWIW, I also found this a surprising... I used `config.env` at first, but no dice.)